### DISCUSSION TITLE
How to disable part of the LED strip?

### DISCUSSION THREAD

**Discussion by Jack2891** [awawa-dev/HyperHDR#330] [General]
Hello community, I have a small problem with the LED setup.

The thing is, everything is working fine, but I used to watch movies with subtitles, and the bottom row of LEDs light up white when subs are on the screen. And to be honest, I don't actually need the bottom row, and instead of physically removing it I would like to disable it.

The bottom row is divided into two sections 33LEDs and 32LEDs. I've tried multiple configurations and only manage to disable one side, the side where the input starts (by moving the input start position) but wasn't able to disable the other end.
Does anyone have an idea how to do it?

![HyperHDR LED count 2](https://user-images.githubusercontent.com/112513447/187531358-00e1a8fb-8edc-45ec-952a-a54ee7629b1f.png)
![HyperHDR LED count 3](https://user-images.githubusercontent.com/112513447/187531359-fc33368f-59a1-48d7-ab7e-932610292c96.png)

**Comment by kesch095** [comment]
Hi, 
I do have the same Feature Request. Would it be possible to implement this? 

Kind regards

**Reply by awawa-dev** [reply]
Hi
Look into the documentation, I think it's implemented.

**Reply by thickbut** [reply]
you can right click on the individual zones in the led preview window for options such as disabling a single zone.

**Reply by kesch095** [reply]
Thank you very much, disabling the LEDs one by one did the trick! :)

Sorry I was just yet able to take another look at it. 

> you can right click on the individual zones in the led preview window for options such as disabling a single zone.
